features/ProfileTab/PreferencesScreen/Currency/CurrencyDropdown.tsx

```diff
@@ -0,0 +1,34 @@
+import { useContext, useState } from "react";
+
+import SettingsContext from "../../../../contexts/SettingsContext";
+import Currency from "../../../../types/Currency";
+
+const data: {
+  label: string;
+  value: Currency;
+}[] = [
+  { label: "Dolar", value: "USD" },
```

Comment:
label: "US Dollar" would be the best option (many countries use their own dollars)

Reply:
Fixed in [fd48dd3](https://github.com/flatly-pw/mobile-app/pull/13/commits/fd48dd317fdca05be37b7145fffa014f31b955d2)